doc/source/ecosystem.rst

```diff
@@ -325,6 +325,17 @@ dimensional arrays, rather than the tabular data for which pandas excels.
 Out-of-core
 -------------
 
+`BCPandas <https://github.com/yehoshuadimarsky/bcpandas>`__
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+BCPandas provides high performance writes from pandas to Microsoft SQL Server,
+far exceeding the performance of the native ``df.to_sql`` method. Internally, it uses
+Microsoft's BCP utility, but the complexity is fully abstracted away from the end user.
+Rigorously tested, it is a complete replacement for ``df.to_sql``.
+
+(Testing has shown that the same peformance speedup is not achieved when reading data,
+only when writing data, so it is not a full I/O replacement, just the "I" part).
```

Comment:
I would leave this out, and just clearly note that it only provides a `to_sql` alternative

Reply:
ok did